src/core/data-tests.js

```diff
@@ -23,11 +23,50 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  const emojiList = document.createElement("span");
+  const [testFile] = new URL(href).pathname.split("/").reverse();
+  const testParts = testFile.split(".");
+  let [testFileName] = testParts;
+
+  const isSecureTest = testParts.find(part => part === "https");
+  if (isSecureTest) {
+    const requiresConnectionEmoji = hyperHTML.bind(
+      document.createElement("span")
+    )` 🔒 `;
+    requiresConnectionEmoji.setAttribute(
+      "aria-label",
+      "requires a secure connection"
+    );
+    requiresConnectionEmoji.setAttribute("title", "Test requires HTTPS");
+    testFileName = testFileName.replace(".https", "");
```

Comment:
Not that previously, you were doing the `replace()` but not assigning it to anything.